test/win-bats/collection.xml

```diff
@@ -1330,6 +1356,25 @@
     set "SAXON_LICENSE=%SAXON_JAR%\..\saxon-license.lic"
     if exist "%SAXON_LICENSE%" call :copy "%SAXON_LICENSE%" "%SAXON_HOME%"
 
+    call :run ..\bin\xspec.bat ^
+        -catalog "catalog\01\catalog-public.xml;catalog\01\catalog-rewriteURI.xml" ^
+        catalog\catalog-01_stylesheet.xspec
+    call :verify_retval 0
+    call :verify_line 18 x "passed: 4 / pending: 0 / failed: 0 / total: 4"
+	</case>
+
+	<case ifdef="BUNDLED_XML_RESOLVER" name="Saxon 11 and later: invoking xspec using SAXON_HOME finds Saxon jar and XML Catalog Resolver jar">
+    set "SAXON_HOME=%WORK_DIR%\saxon %RANDOM%"
+    call :mkdir "%SAXON_HOME%"
+    call :mkdir "%SAXON_HOME%\lib"
+    call :copy "%SAXON_JAR%"              "%SAXON_HOME%"
+    call :copy "%SAXON_JAR%\..\lib\*.jar" "%SAXON_HOME%\lib"
```

Comment:
Like `xspe.bats`, this can be

```suggestion

    call :mkdir "%SAXON_HOME%\lib"
    call :copy "%SAXON_JAR%"        "%SAXON_HOME%"
    call :copy "%SAXON_JAR%\..\lib" "%SAXON_HOME%\lib"
```

Reply:
Fixed in 7f34f8b2b728ce250d5bf7c3d5ea19bd8d78de80